cluster/examples/kubernetes/ceph/sc.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
Could you name the file consistent with other sample storage class yamls? How about `storageclass-csi.yaml`?

Reply:
Also could you add a description to `ceph-examples.md` of this new example?